drivers/include/isl29125.h

```diff
@@ -107,6 +107,37 @@ typedef struct {
 } isl29125_t;
 
 /**
+ * @brief Configuration-3 Register 0x03 B1:0
+ */
+typedef enum {
+    ISL29125_INTERRUPT_STATUS_NONE = 0x00,      /** No interrupt*/
+    ISL29125_INTERRUPT_STATUS_GREEN = 0x01,      /** GREEN interrupt*/
+    ISL29125_INTERRUPT_STATUS_RED = 0x02,      /** RED interrupt*/
+    ISL29125_INTERRUPT_STATUS_BLUE = 0x03      /** BLUE interrupt*/
+} isl29125_interrupt_status_t;
```

Comment:
Please align assignments and comment; and format comments like this:
```
ISL29125_INTERRUPT_STATUS_NONE = 0x00,      /**< No interrupt */
```

with space before closing